tests/humane/client.test.ts

```diff
@@ -0,0 +1,84 @@
+import { describe, it, expect, vi, beforeEach } from 'vitest'
+import { Client } from '@/humane/Client'
+import { Client as ClientT } from '@/api/client/client'
+
+// Mock the API client
+const mockApiClient = {
+    account: {
+        info: vi.fn(),
+    },
+    listPermissions: vi.fn(),
+    listServers: vi.fn(),
+    server: vi.fn(),
+} as unknown as ClientT
```

Comment:
mock client shouldn't be cast to `unknown` and must satisfy ClientT`

```suggestion
const mockApiClient: ClientT = {
    account: {
        info: vi.fn(),
    },
    listPermissions: vi.fn(),
    listServers: vi.fn(),
    server: vi.fn(),
}
```
or

```suggestion
const mockApiClient = {
    account: {
        info: vi.fn(),
    },
    listPermissions: vi.fn(),
    listServers: vi.fn(),
    server: vi.fn(),
} satisfies ClientT
```